js/about/contributionStatement.js

```diff
@@ -328,9 +328,13 @@ class ContributionStatement extends ImmutableComponent {
   }
 
   get ContributionStatementPageFooter () {
+    const l10nDataArgs = {
+      currentYear: new Date().getFullYear().toString(10)
```

Comment:
toString's radix is option so doesn't need to be specified and you can even just not convert to a string since it'll be coerced but I'm fine with this.  Merging, thanks!